docs/release/3.3/release-3-3-34.md

```diff
@@ -6,14 +6,17 @@ OSG Software Release 3.3.34
 Summary of changes
 ------------------
 
+!!! note
+The repositories referenced by osg-release will be retired on May 15th. Update osg-release before that date.
+
+``` console
+root@host # yum update osg-release
```

Comment:
Users will still need to update all the other packages that you reference in your list of `grid.iu.edu` -> `opensciencegrid.org` packages if they have them installed...